docs/automation-example.md

```diff
@@ -0,0 +1,69 @@
+# example workflow
```

Comment:
Name of the workflow? 
Please capitalize the starting letter

Reply:
Done